src/devtools/client/webconsole/actions/input.js

```diff
@@ -121,8 +145,9 @@ function evaluateExpression(expression) {
  *                          devtools/shared/fronts/webconsole.js
  */
 async function evaluateJSAsync(expression, options = {}) {
-  const { asyncIndex, frameId } = options;
-  const rv = await ThreadFront.evaluate(asyncIndex, frameId, expression);
+  const { asyncIndex, frameId, pure } = options;
+  //reminder that there would be no results if the function were impure -logan
+  const rv = await ThreadFront.evaluate(asyncIndex, frameId, expression, pure);
   const { returned, exception, failed } = rv;
 
   let v;
```

Comment:
Can immediately destructure.

Either way, let's please not leave one-letter var names in the code. 😄